src/gui/plugins/scene3d/Scene3D.cc

```diff
@@ -2276,10 +2358,11 @@ void Scene3D::Update(const UpdateInfo &_info,
           return true;
         });
 
-    auto renderWindow = this->PluginItem()->findChild<RenderWindowItem *>();
     renderWindow->SetWorldName(this->dataPtr->worldName);
   }
 
+  msgs::Pose poseMsg = msgs::Convert(renderWindow->CameraPose());
+  this->dataPtr->cameraPosePub.Publish(poseMsg);
```

Comment:
convert the pose and publish only if there are connections?

Reply:
Done in 673ba0fbb1f59383d66ab415a92a768ee1383988